src/modelfree/score_agent.py

```diff
@@ -333,9 +168,12 @@ def score_agent(_run, _seed, env_name, agent_a_path, agent_b_path, agent_a_type,
     def env_fn(i):
         env = make_env(env_name, _seed, i, None, pre_wrapper=pre_wrapper)
         if videos:
-            env = PrettyMujocoWrapper(env, env_name, agent_a_type, agent_a_path,
-                                      agent_b_type, agent_b_path)
-            env = VideoWrapper(env, osp.join(video_dir, str(i)), video_per_episode)
+            if video_params['annotated'] and 'multicomp' in env_name:
```

Comment:
If this will as currently written silently fail to be annotated in the case where it's called with annotated = True but for some reason not a multicomp environment, I think it would be good to throw some kind of warning to that effect

Reply:
added a warning